moduleroot/.github/workflows/ci.yml.erb

```
@@ -29,3 +35,6 @@ jobs:
 <%- if @configs.key?('rubocop') -%>
       rubocop: <%= @configs['rubocop'] %>
 <%- end -%>
+<%- if !Dir[File.join(@metadata[:workdir], 'spec', 'acceptance', '**', '*_spec.rb')].any? && @configs['unit_runs_on'] -%>
```

You can also use `[].none?` instead of `![].any?`.
```suggestion
<%- if Dir[File.join(@metadata[:workdir], 'spec', 'acceptance', '**', '*_spec.rb')].none? && @configs['unit_runs_on'] -%>
```

---

tested it, works fine